src/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,22 +14,17 @@ const ProgressBar = ({ step }: ProgressBarProps) => {
   const { theme, formData } = useContext(Context);
   const totalSteps = getStepDirectory(formData.immutableReferrer).length + STARTING_QUESTION_NUMBER;
   const { id } = useParams();
-  let stepValue: number = 0;
-  if (step !== undefined) {
-    stepValue = step;
-  } else if (id !== undefined) {
-    stepValue = Number(id);
-  }
+
+  let stepValue = step || id || 0;
+  let progressPercentage: number = ((Number(stepValue) - 1) / totalSteps) * 100;
 
   const progressBarStyles = {
     marginBottom: '5px',
-    backgroundColor: '#d6d6d6c4',
-    border: '1px solid #B0B0B0',
+    backgroundColor: '#FBF9FC',
     borderRadius: '500rem;',
     height: '1rem',
-    boxShadow: 'inset -1px 1px 3px rgb(0 0 0 / 0.2)',
     '& .MuiLinearProgress-bar': {
-      background: `linear-gradient(90deg, ${theme.primaryColor} 0%, ${theme.secondaryColor} 100%)`,
+      background: '#D6743F',
```

Comment:
We use a style controller for colors, because we have some partners that want branded colors for people coming from their link. I created am [issue](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/765) to update it. When that gets updated could you use `them.whateverColor` for colors.

Reply:
I'll wait for the issue to be completed to update this accordingly. thanks!

Reply:
Hard-coded colors have been updated to theme colors.

<img width="1459" alt="Screenshot 2023-12-06 at 2 46 21 PM" src="https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/49b3e626-8b96-4198-a0af-a911bb93f02f">

![Screenshot 2023-12-06 at 2 47 04 PM](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/a4b1743c-56ef-468a-bdfd-bc480b981db8)